File: CodeDebuggingKHMWar/CPP/YashikT-Question2.cpp
#include<iostream>
#include<math.h>				// to use pow function
using namespace std;
int main(){
  int num,d=0,n,r,p=0,a=1;
  char choice;
  cin>>num;					//changed insertion operator (<<) to get from operator (>>)
  cin>>choice;					//changed insertion operator (<<) to get from operator (>>)
  switch(choice)				//switch typo fixed
	{
	case 'a':				// added colon, case value should be constant changed it to 'a'
		n=num;
		while(n!=0)
		{
			n=n/10;
			d++;
		}
		n=num;
		while(n!=0)
		{
			r=n%10;
			a=a*pow(r,d);		// sqr changed to pow, removed Equal sign to make it assign
			n=n/10;
		}
		cout<<a<<"\n";  
		break;				// added break
	case 'p':				// added colon, case value should be constant changed it to 'p'
		n=num;
		while(n!=0)
		{
			r=n%10;
			p=(p*10)+r;		// removed Equal sign, changed p+10+r to (p*10)+r
			n=n/10;
		}
		cout<<p<<"\n";
		//continue;			// No need of continue
	}
}

File: CodeDebuggingKHMWar/Python/YashikT-Question2.py
for i in range(int(input())): 				# input() is changed to int(input()), to make it a integer
        word = input()
        word = word.upper() 				# word is assigned to word.upper(),it has a return value
        vowels = ['A','E','I','O','U']
        count = 0 					# removed a Equal Sign to initialize count
        for j in range(1,len(word)):			# added Colon (:), length() is changed to len(); length is not a function
                if word[j] in vowels[:]:		# added Colon inside square brackets, vowels[] to vowels[:]
                        if j == 0: 			# added Equal Sign
                                count += 1		# Changed count ++ to count += 1
                        elif word[j+1] in vowels[:]: 	# added Colon inside square brackets, vowels[] to vowels[:]
                                pass			# change break to pass 
                        else:
                                count += 1		#Changed count ++ to count += 1; python has ++ increment operator
        print (count)
